### PR TITLE
Create snort-installer.sh

### PR DESCRIPTION
**Pull request by berat-eth**

### CODE REVIEW

**Comment by zeroday-software** [comment]
👍

**Comment by ansrb** [comment]
hi. thank you for your submission!

i'm afraid we can't take these changes:
1. they rely on apt installation of snort, which we don't maintain. current snort releases provide source code with build scripts that install it on your machine;
2. it seems that your script is personalized per your setup, judging by some pre-set vars and rules. setup-specific configs are to be set by the end users per their needs, and a generic installer script shouldn't do this;
3. does this script install snort3 or snort2? judging by the config file extension of `.conf` it's snort2, in snort3 we moved onto `.lua` files for configuration. and this is a snort3 repo.

i recognize that this is a working script that should be able to install snort on your system, but this repo isn't really a place for it.

File: snort-installer.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+if [ ! -d "$SNORT_DIR" ]; then
+    echo "Snort directory not found. Please ensure Snort is installed."
+    exit 1
+fi
+
+echo "Backing up the configuration file..."
+sudo cp /etc/snort/snort.conf /etc/snort/snort.conf.bak
+
+echo "Configuring the Snort configuration file..."
+sudo bash -c "echo 'var HOME_NET 192.168.1.0/24' > /etc/snort/snort.conf"
```
Comment (ansrb) [inline snort-installer.sh:19]: `.conf` config file is an indication that this is snort2

**Comment by snortadmin** [comment]
We are not maintaining install foo, even for Snort 3. We will be making docker files available for various distros.

**Pull request closed without merging**